src/lib/libast/include/ast_fs.h

```diff
@@ -0,0 +1,10 @@
+/* : : generated from fs by iffe version 2013-11-14 : : */
+#pragma prototyped
+#ifndef _def_fs_features
+#define _def_fs_features	1
+
+#include <sys/stat.h>
+#include <sys/statvfs.h>
+#define FS_default	"ufs"
```

Comment:
Now is a good time to change this to something like "unknown". Even better, just omit this file and change the one place in *src/lib/libast/string/fmtfs.c* it is used to just hardcode the string "unknown". It should be a can't happen situation so it doesn't really matter what the string is but pretending it's an actual file system type such as "ufs" is not good.